src/nile/utils/status.py

```diff
@@ -0,0 +1,100 @@
+"""Functions used to find/track/debug a transaction status."""
+
+import json
+import logging
+import subprocess
+import time
+from collections import namedtuple
+from enum import Enum
+
+from nile.common import RETRY_AFTER_SECONDS, get_network_parameter_or_set_env
+from nile.utils.debug import debug_message
+
+TransactionStatus = namedtuple(
+    "TransactionStatus", ["tx_hash", "status", "error_message"]
+)
+
+
+def status(
+    tx_hash, network, status_type=None, contracts_file=None
+) -> TransactionStatus:
+    """Fetch a transaction status.
+
+    Optionally track until resolved (accepted on L2 or rejected) and/or
+    use available contracts to help locate the error. Debug implies track.
```

Comment:
```suggestion
    use available artifacts to help locate the error. Debug implies track.
```
I think this is more clear